Sources/Plot/API/HTMLAttributes.swift

```diff
@@ -191,6 +192,18 @@ public extension Node where Context == HTML.AnchorContext {
     }
 }
 
+// MARK: - DateTime
```

Comment:
Code style nitpick: Please leave a blank line between this `MARK` and the following type declaration (as is done elsewhere across the code base)